### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by frapac**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,15 +5,19 @@
...
-Its main features are:
+ExaPF is a HPC package implementing a vectorized modeler
+for power systems. It targets primarily GPU architectures, and provides a portable abstraction to model power system on upcoming HPC architectures.
 
-* Using [CUDA.jl](https://juliagpu.gitlab.io/CUDA.jl/) CuArrays arrays for generating CUDA kernels using the broadcast '.' operator.
-* Using [ForwardDiff.jl](https://github.com/JuliaDiff/ForwardDiff.jl) and Jacobian coloring to generate the compressed Jacobian of the PF equations. The Jacobian evaluation is taking place fully on the GPU.
-* Preconditioned BICGSTAB with support for [Krylov.jl](https://github.com/JuliaSmoothOptimizers/Krylov.jl).
-* A block Jacobi preconditioner that updates on the GPU.
+Its main features are:
+* **Portable approach:** All expressions are evaluated fully on the GPU, without data transfers to the host.
```
Comment (michel2323) [inline README.md:12]: Maybe briefly list what expressions or what is meant by it?

File: README.md [hunk 2]
```diff
@@ -5,15 +5,19 @@
...
+* **Differentiable kernels:** All the expressions are differentiable with [ForwardDiff.jl](https://github.com/JuliaDiff/ForwardDiff.jl). ExaPF uses matrix coloring to generate efficiently the Jacobian and the Hessian in sparse format.
+* **Power flow solver:** ExaPF implements a power flow solver working fully on the GPU, based on a Newton-Raphson algorithm.
+* **Iterative linear algebra:** ExaPF uses [Krylov.jl](https://github.com/JuliaSmoothOptimizers/Krylov.jl) to solve sparse linear systems entirely on the GPU, together with an overlapping Schwarz preconditioner.
 
-This code will serve as the basis for OPF on GPUs using the reduced gradient method. A similar abstraction than CuArrays will be used to port the code to AMD ROCm and Intel oneAPI through [AMDGPU.jl](https://github.com/JuliaGPU/AMDGPU.jl) and [oneAPI.jl](https://github.com/JuliaGPU/oneAPI.jl), respectively.
+ExaPF leverages [KernelAbstractions.jl](https://github.com/JuliaGPU/KernelAbstractions.jl)
+to generate portable kernels working on different backends.
+Right now, only CUDA is fully supported, but in the medium term we have good hope to support
```
Comment (michel2323) [inline README.md:19]: As long as there's hope.

File: README.md [hunk 3]
```diff
@@ -29,26 +33,36 @@ pkg> test ExaPF
 
 ### How to solve the power flow of a given MATPOWER instance?
 
-ExaPF implements a Newton-Raphson algorithm to solve
-the power flow equations of a power network.
+ExaPF solves the power flow equations of any power network with a Newton-Raphson algorithm:
```
Comment (michel2323) [inline README.md:36]: Any? Maybe `a`.

File: README.md [hunk 4]
```diff
@@ -29,26 +33,36 @@ pkg> test ExaPF
...
+  * Time linear solver (s) ...: 0.0010
+  * Time total (s) ...........: 0.0014
 ```
 
+For more information on how to solve power flow on the GPU,
+please refer to the [quickstart guide](https://exanauts.github.io/ExaPF.jl/dev/quickstart/).
+
+## Extensions
+
+- [Argos.jl](https://github.com/exanauts/Argos.jl/) uses ExaPF as a modeler to accelerate the resolution of OPF problems on CUDA GPU.
```
Comment (michel2323) [inline README.md:64]: Nice

File: README.md [hunk 5]
```diff
@@ -63,6 +77,7 @@ This research was supported by the Exascale Computing Project (17-SC-20-SC), a j
 
 [docs-stable-img]: https://img.shields.io/badge/docs-stable-blue.svg
 [docs-stable-url]: https://exanauts.github.io/ExaPF.jl/
+[docs-stable-url]: https://exanauts.github.io/ExaPF.jl/
```
Comment (michel2323) [inline README.md:80]: Duplicate. Maybe you meant something else.

**Review by michel2323 — APPROVED** [review]

**Comment by frapac** [comment]
Thanks for the feedback Michel!

**Pull request merged**